neuromation/api/quota.py

```diff
@@ -42,17 +42,24 @@ def __init__(self, core: _Core, config: Config) -> None:
             return _quota_info_from_api(res)
 
 
-def _quota_info_from_api(payload: Dict[str, Any]) -> _QuotaInfo:
-    jobs = payload["jobs"]
-    spent_gpu = float(int(jobs["total_gpu_run_time_minutes"]) * 60)
-    spent_cpu = float(int(jobs["total_non_gpu_run_time_minutes"]) * 60)
-    quota = payload["quota"]
-    limit_gpu = float(quota.get("total_gpu_run_time_minutes", "inf")) * 60
-    limit_cpu = float(quota.get("total_non_gpu_run_time_minutes", "inf")) * 60
-    return _QuotaInfo(
-        name=payload["name"],
-        gpu_time_spent=spent_gpu,
-        gpu_time_limit=limit_gpu,
-        cpu_time_spent=spent_cpu,
-        cpu_time_limit=limit_cpu,
-    )
+def _quota_info_from_api(payload: Dict[str, Any]) -> Dict[str, _QuotaInfo]:
+    clusters = payload["clusters"]
+    ret: Dict[str, _QuotaInfo] = {}
+    for cluster_payload in clusters:
+        cluster_name = cluster_payload["name"]
+        jobs_payload = cluster_payload["jobs"]
+        quota_payload = cluster_payload.get("quota", {})
+        ret[cluster_name] = _QuotaInfo(
+            cluster_name=cluster_name,
+            cpu_time_spent=float(
+                int(jobs_payload["total_non_gpu_run_time_minutes"]) * 60
```

Comment:
This was in the current code, but why do we need to call `int()` and `float()`? Are not all these values already integers?

Reply:
Good catch! Will change soon

Reply:
@serhiy-storchaka , can you check latest implementation?